src/components/vm/vmReplaceSpiceDialog.jsx

```diff
@@ -74,16 +123,21 @@ export const ReplaceSpiceDialog = ({ vmName, vmId, connectionName, vmRunning })
                    </Button>
                </>
            }>
-            { vmRunning && !error && <NeedsShutdownAlert idPrefix="spice-modal" /> }
+            { vm.state === 'running' && !error && <NeedsShutdownAlert idPrefix="spice-modal" /> }
             {error && <ModalError dialogError={error.dialogError} dialogErrorDetail={error.dialogErrorDetail} />}
             <TextContent>
-                <Text>{_("Reconfigure the virtual machine for a host which does not support SPICE, for host upgrades or live migration:")}</Text>
+                <Text>{
+                    isMultiple
+                        ? _("Reconfigure the selected virtual machines for a host which does not support SPICE, for host upgrades or live migration:")
+                        : _("Reconfigure the virtual machine for a host which does not support SPICE, for host upgrades or live migration:")
+                }</Text>
                 <TextList>
                     <TextListItem>{_("Convert SPICE graphics console to VNC.")}</TextListItem>
                     <TextListItem>{_("Convert QXL video card to VGA.")}</TextListItem>
                     <TextListItem>{_("Remove SPICE audio and host devices.")}</TextListItem>
```

Comment:
Nitpick: For lists with fragment phrases, you don't use periods, unless there are two separate thoughts, then you'd use some kind of punctuation to separate them and apply it to all (to stay consistent).

For this list, we shouldn't use periods for any of them.

```suggestion
                    <TextListItem>{_("Convert SPICE graphics console to VNC")}</TextListItem>
                    <TextListItem>{_("Convert QXL video card to VGA")}</TextListItem>
                    <TextListItem>{_("Remove SPICE audio and host devices")}</TextListItem>
```

PatternFly doesn't touch on this, except for headings and titles @ https://www.patternfly.org/ux-writing/punctuation — but it's a general thing when writing, even outside of computers. https://proofed.com/writing-tips/should-bullet-point-lists-end-with-a-period/

It's technically not wrong, as-is, but it's still best to leave out periods for UIs, unless needed.

Reply:
Done, thanks!